api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/LocationTranslatorImplTest.java

```diff
@@ -271,4 +275,29 @@ public void getLocationContactDetails_shouldWorkAsExpected() {
 		assertThat(omrsLocation.getActiveAttributes().size(), equalTo(1));
 		
 	}
+	
+	@Test
+	public void toFhirResource_shoudTranslateOpenmrsTagsToFhirLocationTags() {
+		LocationTag tag = new LocationTag("Login location", "Used to identify login locations");
+		omrsLocation.addTag(tag);
+		org.hl7.fhir.r4.model.Location fhirLocation = locationTranslator.toFhirResource(omrsLocation);
+		assertThat(fhirLocation.getMeta().getTag(), notNullValue());
+		assertThat(fhirLocation.getMeta().getTag().size(), greaterThanOrEqualTo(0));
+		assertThat(fhirLocation.getMeta().getTag().get(0).getCode(), is("Login location"));
+	}
+	
+	@Test
+	public void toOpenmrsType_shouldTranslateFhirTagsToOpenmrsLocationTags() {
+		List<Coding> tags = new ArrayList<>();
+		Coding tag = new Coding();
+		tag.setCode("Lab location");
+		tag.setDisplay("Used to identify lab resources");
+		tags.add(tag);
+		org.hl7.fhir.r4.model.Location fhirLocation = new org.hl7.fhir.r4.model.Location();
+		fhirLocation.getMeta().setTag(tags);
+		omrsLocation = locationTranslator.toOpenmrsType(fhirLocation);
+		assertThat(omrsLocation.getTags(), notNullValue());
+		assertThat(omrsLocation.getTags().size(), greaterThanOrEqualTo(0));
```

Comment:
I think the size here should be greaterThanOrEqualTo(1):
    `assertThat(omrsLocation.getTags().size(), greaterThanOrEqualTo(1));`

Reply:
You could even use `assertThat(omrsLocation.getTags(), hasSize(greaterThanOrEqualTo(1));`, but that might be getting too fancy.